components/dashboard/script.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+
+const APP_NAME = "Gitpod";
+
+const vars = {
+    REACT_APP_HOSTNAME : APP_NAME,
+    REACT_APP_DOCS_URL : "https://www.gitpod.io/docs/",
+    REACT_APP_SOCIAL_URL : "https://community.gitpod.io/",
+    REACT_APP_FAVICON_192 : "./favicon192.png",
+    REACT_APP_FAVICON_256 : "./favicon256.png",
+    REACT_APP_FAVICON_512 : "./favicon512.png",
+    REACT_APP_ICON_1 : "./icons/gitpod.svg",
+    REACT_APP_ICON_2 : "../icons/gitpod.svg",
+    REACT_APP_BRNAD_SVG : "./images/gitpod.svg"
```

Comment:
<details>
<summary>:warning: <strong>This code is using `"` double-quotes to declare strings.</strong> (click for details)</summary>

A possible improvement would be to use `'` single-quote to declare all strings.  
  
This change would make the code consistent and easier to maintain.

**[More details](https://confluence.devfactory.com/x/8BhnFQ)**

> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="d2c89e5f-462f-42e4-86cf-4c60ce43f701" data-finding-id="359078128" href="none" codereview-uuid-reserved-tag data-business-id="ee52759d-ffcd-4802-9ba9-923c2189015b" data-analyzer="js" />)

Reply:
<details>
<summary>:warning: <strong>Formatting related to spacing is inconsistent.</strong> (click for details)</summary>

A possible improvement would be to apply the current language Coding Conventions (or project specific style) to all files and fix any possible spacing issue before committing to maintain code style consistency.  
  
This change would ensure that the code is consistently clean, thus, improving readability, making variable and block scopes easier to understand.

**[More details](https://confluence.devfactory.com/x/bHvlFQ)**

## The improvement suggestion in this case
```js  
REACT_APP_BRNAD_SVG: './images/gitpod.svg'
```  

> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="a04a592a-2b92-4b30-97af-89ec46b7743b" data-finding-id="367360876" href="none" codereview-uuid-reserved-tag data-business-id="ee52759d-ffcd-4802-9ba9-923c2189015b" data-analyzer="js" />)